mpv.py

```diff
@@ -479,6 +479,12 @@ def command(self, name, *args):
                 for arg in args if arg is not None ] + [None]
         _mpv_command(self.handle, (c_char_p*len(args))(*args))
 
+    def play(self):
+        self._set_property("pause", False, proptype=bool)
+
+    def pause(self):
```

Comment:
This name collides with the ```pause``` property that is mapped into the MPV object as an attribute. I don't have a good idea how to solve this naming conflict right now. I would rather preserve the property's name for API consistency.
```python
import mpv
p = mpv.MPV()
p.pause = True
p.pause = False
```